### PR TITLE
Feature Initialize Frontend: Init react source

### PR DESCRIPTION
**Pull request by maituan94**

### CODE REVIEW

File: client/src/_global.scss [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    color: white;
+}
+
+.header {
+    position: fixed;
+    width: 100%;
+    top: 0;
+}
+
+.toast-container{
```
Comment (ranmeetSingh8780) [inline client/src/_global.scss]: This is unused. Should be removed.

File: client/src/_global.scss [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+    }
+    input[type="radio"]:checked, input[type="checkbox"]:checked  {
+        border-color: black;
+        background-color: black;
+    }
+}
+
+.navbar-nav {
+    cursor: pointer;
+}
```
Comment (MoondKartik) [inline client/src/_global.scss]: There is no styling in the source initialization.
Please help remove

**Review by MoondKartik — COMMENTED** [review]
Please check my review

**Review by ranmeetSingh8780 — COMMENTED** [review]
Please check my comment

File: client/src/_global.scss (hunk 1 above)
Comment (maituan94) [reply client/src/_global.scss]: Resolved.

File: client/src/_global.scss (hunk 2 above)
Comment (maituan94) [reply client/src/_global.scss]: I removed the styling.

**Review by ranmeetSingh8780 — APPROVED** [review]
I approved the pr.

**Pull request merged**